lib/naughty/core.lua

```diff
@@ -583,6 +583,7 @@ function naughty.notify(args)
             die(naughty.notificationClosedReason.dismissedByUser)
         end
     end
+    notification.run = run
```

Comment:
I think it's useful when you want to call actions for notifications sent through dbus from other apps. For example, we have notificaton sent from some app with some action in run argument (can be triggered by clicking on it) and also several actions in actions argument. I want to be able to draw menu by hotkey (when is see notification on the screen) with all these actions and call some of them using my keyboard. I can get notification from naughty.notificatoins table, but don't know other way to get available actions from it